galsim/nfw_halo.py

```diff
@@ -42,7 +42,15 @@ def __init__(self, omega_m=0.3, omega_lam=0.7):
         self.omega_lam = omega_lam
         self.omega_c = (1. - omega_m - omega_lam)
         #self.omega_r = 0
```

Comment:
There are other places that would also have to change, but don't have comments.  So it seemed a bit inconsistent to me.  But I guess it doesn't matter, we can cross that bridge when/if we come to it.  :)